### DISCUSSION TITLE
how to build a three-dimensional model with a spherical grain in the center using neper?

### DISCUSSION THREAD

**Discussion by vicLzs** [neperfepx/neper#675] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/57779487/229261581-9311c4a9-95a2-4bc3-9e56-f34d9f184f5a.png)
As shown in the figure, how to build a three-dimensional model with a spherical grain in the center. There are two grains in this model.

**Comment by smallrain1996** [comment, marked as answer]
I can categorically say that this technology is not possible in the current Neper.

**Reply by rquey** [reply]
Correct - Neper does not generate this type of morphology.
